kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -96,6 +96,35 @@
   const UserKinds = require('kolibri.coreVue.vuex.constants').UserKinds;
 
   module.exports = {
+    $trNameSpace: 'user-edit-modal',
+    $trs: {
+      modalTitle: 'Edit Account Info',
+      // input labels
+      fullName: 'Full Name',
+      username: 'Username',
+      userKind: 'User Kind',
+      enterNewPw: 'Enter new password',
+      confirmNewPw: 'Confirm new password',
```

Comment:
We are on the same page, we can do a consistency check closer to the final release.
And those MD punctuation conventions sounded weird to me too!